src/devices/Cargo.toml

```diff
@@ -14,6 +14,6 @@ net_gen = { path = "../net_gen" }
 polly = { path = "../polly" }
 rate_limiter = { path = "../rate_limiter" }
 snapshot = { path = "../snapshot" }
-versionize = { path = "../versionize" }
-versionize_derive = { path = "../versionize_derive" }
+versionize = { git = "https://github.com/firecracker-microvm/versionize" }
```

Comment:
We need to add tags/releases on all the crates. Otherwise, every change in versionize is automatically pulled in firecracker and this can break the firecracker build.

Reply:
Yup, I opened this early to get coverage numbers and early feedback. I will mark it as RFC to be clear.

Reply:
We also need a changelog in versionize/versionize_derive.

Reply:
done